specification/dimensions/usagequantity.md

```diff
@@ -0,0 +1,30 @@
+# Usage Quantity
+
+Usage Quantity is the amount of resources or services consumed. The Usage Quantity is measured in units specified by the Usage Unit. Usage Quantity may differ from a Pricing Quantity when providers use a different unit of measure for calculating cost.Usage Quantity is often used when analyzing environments to gain understanding of usage/consumption of given products/services in an environment.
```

Comment:
Usage Quantity is the amount of resources or services consumed. The Usage Quantity is measured in units specified by the Usage Unit. Usage Quantity may differ from a Pricing Quantity when providers use a different unit of measure for calculating cost. Usage Quantity is often used when analyzing environments to gain understanding of usage/consumption of given products/services in an environment.